docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `organizationName`       | MUST         | The CA's name or DBA. The CA MAY include information in this field that differs slightly from the verified name, such as common variations or abbreviations, provided that the CA documents the difference and any abbreviations used are locally accepted abbreviations; e.g. if the official record shows "Company Name Incorporated", the CA MAY use "Company Name Inc." or "Company Name". | [Section 3.2.2.2](#3222-dbatradename) |
+| `organizationalUnitName` | SHOULD NOT   | __**TBD**__ |                |
+| `commonName`             | MUST         | The contents SHOULD be an identifier for the certificate such that the certificate's Name is unique across all certificates issued by the issuing certificate. | |
+| Any other attribute      | SHOULD NOT   | __**TBD**__ |                |
+
+##### 7.1.2.8.2 Authority Information Access
+
+If present, the `AuthorityInformationAccesssSyntax` MUST contain one or more `AccessDescription`s. Each `AccessDescription` MUST only contain a permitted `accessMethod`, as detailed below, and each `AuthorityInformationAccessSyntax` MUST contain all required `AccessDescription`s.
+
+| __Access Method__ | __OID__            | __Presence__ | __Access Location__         | __Description__ |
+| --                | --                 | -            | ----                        | ---             |
+| `id-ad-caIssuers` | 1.3.6.1.5.5.7.48.2 | SHOULD       | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's certificate. |
+| `id-ad-ocsp`      | 1.3.6.1.5.5.7.48.1 | MAY          | `uniformResourceIdentifier` | An HTTP URL of the Issuing CA's OCSP responder. |
+| Any other value   | -                  | MUST NOT     | -                           | -               |
+
+##### 7.1.2.8.3 Basic Constraints
+
+| __Field__           | __Description__ |
+| ---                 | ------- |
+| `cA`                | MUST be set TRUE |
+| `pathLenConstraint` | MAY be present |
+
+##### 7.1.2.8.4 Certificate Policies - Affiliated CA
+
+The following requirements apply to the Certificate Policies extension within CA certificates that are issued to and operated by an organization that is the same as the organization operating the Issuing CA or that is an Affiliate of the organization operating the Issuing CA.
+
+If present, the Certificate Policies extension MUST be one of the following two formats:
+
+Table: No Policy Restrictions
+
+| __Field__                  | __Presence__ | __Description__                    |
+| ---                        | -            | ------                             |
+| `certificatePolicies`      |              |                                    |
+| \ \ **1**                  |              | The first `PolicyInformation` present in the `certificatePolicies`. |
+| \ \ \ \ `policyIdentifier` | MUST         | The `anyPolicy` (OID: 2.5.29.32.0) identifier. |
+| \ \ \ \ `policyQualifiers` | MUST NOT     |                                    |
```

Comment:
RFC 5280 has long specifically and explicitly discouraged this, with a SHOULD NOT, which is conveyed here as a MUST NOT.

> To promote interoperability, this profile RECOMMENDS that policy information terms consist of only an OID.

Reply:
I'm not sure how others feel about this but I always thought it is useful to find a CPS URL in CA Certificates pointing me to the issuing CA and read the policies and practices how that intermediate CA Certificate was issued and how this Subordinate CA works.
Obviously there must be a lot of background information and discussions in IETF that I'm not aware of, but a quick search revealed that this recommendation from RFC 5280 is not followed by the majority of publicly-trusted CAs. If the SCWG agrees on a MUST NOT, it should have a future effective date.

Reply:
This is an existing BR restriction though. The current language in the BRs is hopefully clear about this:

https://github.com/cabforum/servercert/blame/9672b03bec91ad9a80f826e928e47f5c1f82964b/docs/BR.md#L1778-L1786
```markdown
The following fields MAY be present if the Subordinate CA is not an Affiliate of the entity that controls the Root CA.

  * `certificatePolicies:policyQualifiers:policyQualifierId` (optional)
```

This requirement is within the affiliated CA portion, and so since the CA does not meet the MAY requirement, it's not permitted.

(This is, after all, what profiles are trying to better clarify :P )

Reply:
Just circling back on this: Does the related discussion with @wthayer  in https://github.com/sleevi/cabforum-docs/pull/36#discussion_r867202774 help clarify the expectations here?

Per our recent validation call, it sounded like we coalesced on "Allow `cPSUri` for everything" (i.e. removing the Affiliate restriction), but there's still a question of the expected contents (issuer CA or subject CA) and, especially for anyPolicy, whether this is at all sensible.